part-one-answer.js

```diff
@@ -0,0 +1,17 @@
+function showresult(val){
+    var n1=parseFloat(document.getElementById('num1').value);
+    var n2=parseFloat(document.getElementById('num2').value);
+    var r=n1+n2;
+    /*var c=val;
+    
+    switch(c)
+        {
+        case '1':
+            r=n1+n2;
+            break;
+        default:
+            break;     
+        }*/
```

Comment:
Looks like you were messing around with next week's assignment. Always delete unused code before committing so it doesn't clutter the code base